lib/assets/javascripts/turbograft.js

```diff
@@ -0,0 +1,37 @@
+//= require_self
+//= require_tree ./turbograft
+
+if (window.TurboGraft == null) { window.TurboGraft = { handlers: {} }; }
+
+TurboGraft.tgAttribute = function(attr) {
+  if (attr.slice(0, 3) === 'tg-') {
+    return `data-${attr}`;
+  } else {
+    return `data-tg-${attr}`;
```

Comment:
https://caniuse.com/?search=template%20string

FYI This isn't supported by all browsers, and neither is `class` and some of ES6 features we're using. Though, given our preference to use import maps, I imagine we don't care about these edge cases.

Reply:
ES6 is compiled to ES5 by sprockets in the final result. Those assets are never returned to users as is.